src/layouts/BaseLayout.astro

```diff
@@ -31,7 +31,7 @@ const { title, description, image, article, publishDate, author } = Astro.props;
       author={author}
     />
   </head>
-  <body class="flex flex-col min-h-screen bg-white text-gray-900 antialiased">
```

Comment:
I don't think we should change these colors just yet. There is a better way to globally adjust color schemes that I can walk you guys through